pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -3466,6 +3463,9 @@ public boolean isSizeBacklogExceeded() {
 
     @Override
     public long getBestEffortOldestUnacknowledgedMessageAgeSeconds() {
+        if (!hasBacklogs(false)) {
```

Comment:
What is the performance impact of this change? Is that a valid concern?

Reply:
Here, use `getPreciseBacklog = false`, it will use two variables calc backlogs, I guess no performance impact.

https://github.com/apache/pulsar/blob/2c3909c17b0c68a39f2f6f2a50c19216ef3a6ffc/managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java#L1132

Do you have any specific considerations?

Reply:
That's good. Just confirming that it's not creating requests to bookkeeper each time.